MatrixKit/Controllers/MXKRoomViewController.m

```diff
@@ -147,6 +147,11 @@ The identifier of the current event displayed at the bottom of the table (just a
     BOOL restartConnection;
 }
 
+@property NSString *openedAttachmentEventId;
+@property NSString *closedAttachmentEventId;
```

Comment:
Please add comments for these 3 new properties.
To be honest, I don't understand the meaning of `closedAttachmentEventId`.